azure/scope/managedcontrolplane_test.go

```diff
@@ -467,6 +468,12 @@ func TestManagedControlPlaneScope_OSType(t *testing.T) {
 	}
 }
 
+type fakeVnetDescriber struct{}
+
+func (f fakeVnetDescriber) IsManaged(ctx context.Context) (bool, error) {
+	return false, errors.New("The subscription '00000000-0000-0000-0000-000000000000' could not be found.")
```

Comment:
even though this is closer to what was happening before, I'm not sure this is actually what the test intends to test. The fact that we're swallowing the error is not great, but probably not in scope for this PR...

Reply:
Agreed, I think the SDKv1 version of this test wasn't intentionally following this path, just getting away with it. I can drop the error if you prefer.

Reply:
I'm +1 for dropping the error

If we want to test the error scenario that should be a separate test case but not sure that's necessary since we're not handling the error, just logging it

Reply:
Done.